.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,37 @@
+name: CI Tests
+
+on:
+  pull_request:
+  push:
+    branches:    
+      - master
```

Comment:
Why would we need to run tests on push to master too? Master branch is protected from direct pushes and we can set branches to be required to be up to date.

Reply:
I think that's to catch regressions but, yeah, we could require branches to be rebased.

The monorepo runs CI on merges to master too. Maybe we should turn that off?